disnake/embeds.py

```diff
@@ -65,6 +65,12 @@ def __repr__(self) -> str:
     def __getattr__(self, attr: str) -> None:
         return None
 
+    def __eq__(self, other: Any) -> bool:
+        return isinstance(other, EmbedProxy) and self.__dict__ == other.__dict__
+
+    def __ne__(self, other: Any) -> bool:
+        return not self.__eq__(other)
```

Comment:
```suggestion
```
This isn't necessary, `__ne__` automatically calls `not __eq__` if it's not implemented: https://docs.python.org/3.10/reference/datamodel.html?highlight=__ne__#object.__ne__